examples/fluid/pipe_flow_2d.jl

```diff
@@ -77,11 +77,15 @@ fluid_system = EntropicallyDampedSPHSystem(pipe.fluid, smoothing_kernel, smoothi
 
 # ==========================================================================================
 # ==== Open Boundary
-function velocity_function(pos, t)
-    # Use this for a time-dependent inflow velocity
-    # return SVector(0.5prescribed_velocity * sin(2pi * t) + prescribed_velocity, 0)
-
-    return SVector(prescribed_velocity, 0.0)
+if !@isdefined(VELOCITY_FUNC)
+    # to avoid double declaration with `trixi_include`
+    function velocity_function2d(pos, t)
+        # Use this for a time-dependent inflow velocity
+        # return SVector(0.5prescribed_velocity * sin(2pi * t) + prescribed_velocity, 0)
+
+        return SVector(prescribed_velocity, 0.0)
+    end
+    const VELOCITY_FUNC = true
 end
```

Comment:
What? Why? Works for me without that stuff.

Reply:
I can run this file and the 3D file without issues without this change here.

Reply:
Yes I can as well but it fails the CI

Reply:
Other option would be to suppress the function duplication errors but this seems to me wrong

Reply:
and change in the 2D example the following:
```julia
n_buffer_particles = 4 * pipe.n_particles_per_dimension[2]^(ndims(pipe.fluid) - 1)
```

```julia
plane_in = ([0.0, 0.0], [0.0, domain_size[2]])
inflow = InFlow(; plane=plane_in, flow_direction,
                open_boundary_layers, density=fluid_density, particle_spacing)
```

Reply:
What did you change?

Reply:
He removed the first include. And just added the definitions to the 3d file.

Reply:
Ah now I understand the problem. It's a function re-definition warning that is only shown in CI with additional warnings. I'm fine with ignoring this if all other options make the example file ugly.